src/layouts/BaseLayout.js

```diff
@@ -195,11 +193,7 @@ const breadcrumbNameMap = {
     title: 'List of adjusted resources',
   },
   '/resources/items': { name: 'Items', title: 'List of available items' },
-  '/resources/warehouses': {
-    name: 'Warehouses',
-    title: 'List of available warehouses',
-  },
-  '/resources/items-categories': {
+  '/resources/item-categories': {
```

Comment:
can you update this to itemcategories since it is the same conventions used on all url path